_schemaV2.graphql

```diff
@@ -13687,6 +13709,9 @@ type Mutation {
 
   # Add (or remove) an artwork to (from) a users dislikes.
   dislikeArtwork(input: DislikeArtworkInput!): DislikeArtworkPayload
+
+  # Updates a Task on the logged in User
+  dismissTask(input: DismissTaskMutationInput!): DismissTaskMutationPayload
```

Comment:
Is it possible to make the payload and all types downstream non-nullable?

Reply:
Added here https://github.com/artsy/metaphysics/pull/6078/commits/0c04577773226e9688d035080d4238c160b2da91
For the payload itself I don't see we are following that pattern with the other mutations